src/components/provider/component_defaults/component_defaults.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { createContext, useContext, FunctionComponent } from 'react';
+
+import { EuiPortalProps } from '../../portal';
+
+export type EuiComponentDefaults = {
+  /**
+   * Provide a global setting for EuiPortal's default insertion position.
+   */
+  EuiPortal?: { insert: EuiPortalProps['insert'] };
```

Comment:
```suggestion
  EuiPortal?: Partial<Pick<EuiPortalProps, 'insert'>>;
```
What do you think of doing this instead with the future goal of replacing it with just `Partial<EuiPortalProps>` when we're ready to support all props?

Reply:
I'm hesitant to expand this so soon, I think I'd rather stick to specific usages for now (per our beta flag) and expand it later when we decide to.

If a consumer really wants to they could `@ts-ignore` and pass in other global EUI props in any case and that would work; so the support is technically there, but I don't want to be explicit about it just yet. Also, EuiPortal doesn't really have any other props (options are `children` and a ref function) that make sense to be configured globally vs per-instance in any case 😅

Reply:
@tkajtoch - just wanted to give you a heads up that I missed the Pick in this code suggestion and you were totally right on it 🤦 and I implemented it in a later PR here: https://github.com/elastic/eui/commit/c4bd1b10c3b6aee26697a35f30c4ea9f427ac5be